pkg/reconciler/instances/eventing/preaction/removenatsoperatorstep.go

```diff
@@ -90,10 +93,54 @@ func (r *removeNatsOperatorStep) removeNatsOperatorResources(context *service.Ac
 	// set the right eventing name, which went lost after rendering
 	manifest.Manifest = strings.ReplaceAll(manifest.Manifest, natsSubChartPath, eventingNats)
 
+	clientSet, err2 := kubeClient.Clientset()
+	if err2 != nil {
```

Comment:
(nit) why not reuse `err` instead of `err2`?

Reply:
linter was complaining because of the ineffective usage of the `err`, which was already initialised before in the code